### ISSUE TITLE
The training codes of DCGAN to train CIFAR10

### ISSUE THREAD

**Issue by LiuzhuForFun** [kwotsin/mimicry#8]
Thanks for this excellent work！
But I met one problem to train DCgan on CIFAR10. Could you please to give me some details of this configs of training?

**Comment by kwotsin** [comment]
Hi @LiuzhuForFun, thanks for your feedback! For training DCGAN, I follow the original configurations for the reported score [1], and specifically from Table 10 and section 2.6 of [1]:
- Learning rate: 2e-4 for D and G.
- (beta_1, beta_2) = (0.5, 0.999) for Adam optimizer
- 5 discriminator updates per generate update
- Train for 200k steps, no learning rate scheduling.
- Batch size 64

Note that the model uses [dcgan_cifar.py](https://github.com/kwotsin/mimicry/blob/master/torch_mimicry/nets/dcgan/dcgan_cifar.py) and the non-saturating loss, following the original configuration

Hope it helps! :D

[1] https://arxiv.org/pdf/1807.04720.pdf
